modules/understanding-upgrade-channels.adoc

```diff
@@ -7,159 +7,146 @@
+upgrade channels will never include an upgrade to a 4.5 release. This strategy ensures that
+administrators explicitly decide to upgrade to the next minor version of
+{product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the `openshift-install`
+binary file for a specific version of {product-title} always installs that version.
 
-{product-title} {product-version} has the following upgrade channels to choose from:
+{product-title} {product-version} offers the following upgrade channels:
 
-* candidate-{product-version}
-* fast-{product-version}
-* stable-{product-version}
+* `candidate-{product-version}`
+* `fast-{product-version}`
+* `stable-{product-version}`
 
 [discrete]
-== candidate-{product-version} Channel
+== candidate-{product-version} channel
 
-The candidate-{product-version} channel will contain candidate builds for a z-stream
+The `candidate-{product-version}` channel contains candidate builds for a z-stream
 ({product-version}.z) release.
-Release candidates contain all the features of the product but they are not supported and
-should only be used to test feature acceptance and assist in qualifying the next version
+Release candidates contain all the features of the product but are not supported. Use release candidate versions to test feature acceptance and assist in qualifying the next version
 of {product-title}.
-A release candidate is any build (e.g. 4.3.0-rc.3, 4.3.0) that is available in the candidate
+A release candidate is any build, such as `4.3.0-rc.3` that is available in the candidate
 channel.
-After a version lands in the candidate channel, it goes through more quality checks and if
-it meets the quality standard it is promoted to fast-{product-version} or stable-{product-version}
-channels.
-So if a given release is available in the candidate-{product-version} channel and also in the fast-{product-version}
-or stable-{product-version} channels, it is a Red Hat supported version.
-Additionally candidate-{product-version} may include dead end releases from which there are no or ever
-be recommended upgrades.
+After a version is available in the candidate channel, it goes through more quality checks. If
+it meets the quality standard, it is promoted to the `fast-{product-version}` or `stable-{product-version}` channels.
+Because of this strategy, if a specific release is available in both the `candidate-{product-version}` channel and in the `fast-{product-version}`
+or `stable-{product-version}` channels, it is a Red Hat supported version.
+The `candidate-{product-version}` channel can include release versions from which there are no recommended updates in any channel.
 
-The candidate-{product-version} channel also allows upgrading from a previous minor version of
-{product-title} to {product-version}.
+You can also use the `candidate-{product-version}` channel to upgrade from a previous minor version of
```

Comment:
nit: Drop "also"?  You don't include it in the analogous `fast-...` section below.